pkg/services/org/orgimpl/store.go

```diff
@@ -822,3 +832,8 @@ func removeUserOrg(sess *db.Session, userID int64) error {
 	_, err := sess.ID(userID).MustCols("org_id").Update(&user)
 	return err
 }
+
+// RegisterDelete registers a delete query to be executed when an org is deleted, used to delete enterprise data.
+func (ss *sqlStore) RegisterDelete(query string) {
```

Comment:
Is it introduced for a later need?

Reply:
It's being used to delete team sync entries: https://github.com/grafana/grafana-enterprise/pull/5912